pac4j-http/src/main/java/org/pac4j/http/credentials/authenticator/IpRegexpAuthenticator.java

```diff
@@ -46,9 +53,19 @@ public void validate(final TokenCredentials credentials) {
         if (!this.pattern.matcher(ip).matches()) {
             throw new CredentialsException("Unauthorized IP address: " + ip);
         }
+
+        final IpProfile profile = new IpProfile(ip);
```

Comment:
The profile should be saved in the credentials as the default profile creator gets it there.